patches/configure.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/configure b/configure
+index 7bee4db..2476128 100755
+--- a/configure
++++ b/configure
+@@ -5786,6 +5786,10 @@ aix[4-9]*)
+   lt_cv_deplibs_check_method=pass_all
+   ;;
+ 
++openedition)
++  lt_cv_deplibs_check_method=pass_all
++  ;;
++ 
+ beos*)
+   lt_cv_deplibs_check_method=pass_all
+   ;;
+@@ -14109,6 +14113,7 @@ $as_echo "$as_me: WARNING:
+ fi
+ 
+ fi
++
+ if test "$use_descriptor_passing" = "yes"; then
+ 
+ $as_echo "#define USE_DESCRIPTOR_PASSING 1" >>confdefs.h
+@@ -14174,6 +14179,7 @@ else
+   ;;
+   *)
+   as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
++
+ for as_dir in $PATH
+ do
+   IFS=$as_save_IFS
+@@ -14293,9 +14299,9 @@ $as_echo "$as_me: Use gpgrt-config with $gpgrt_libdir as gpg-error-config" >&6;}
+     req_minor=`echo $min_gpg_error_version | \
+                sed 's/\([0-9]*\)\.\([0-9]*\)/\2/'`
+     major=`echo $gpg_error_config_version | \
+-               sed 's/\([0-9]*\)\.\([0-9]*\).*/\1/'`
++               sed 's/\([0-9]*\)\.\([0-9]*\).*/\1/' | cut -f 3 -d' ' `
+     minor=`echo $gpg_error_config_version | \
+-               sed 's/\([0-9]*\)\.\([0-9]*\).*/\2/'`
++               sed 's/\([0-9]*\)\.\([0-9]*\).*/\2/' | cut -f 3 -d' '`
```

Comment:
can you elaborate on why we need a cut?

Reply:
Configure steps was failing without this fix, because required major and minor values were supposed to be only digits but, the value on echo was printing :
**major = gpg-error (libgpg-error) 1 , minor = gpg-error (libgpg-error) 47**. This error is also seen on linux machine. 

With this fix we get **major = 1 , minor = 47** and now these are actual major and minor versions required , and configure step proceeds without any issues after this.